pkg/kev/testdata/init-default/docker-compose-yml/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.9'
+version: '2.4'
```

Comment:
Why this change?

Reply:
`restart: always` is a v2 docker-compose type config not v3.

> deploy.restart_policy  ... Configures if and how to restart containers when they exit. Replaces restart.
See here: https://docs.docker.com/compose/compose-file/compose-file-v3/#restart_policy

Reply:
see my comment below. `Restart` is still a valid v3 field.